.golangci.yaml

```diff
@@ -55,6 +55,10 @@ linters-settings:
     alias:
       - pkg: github.com/kyma-project/template-operator/api/v1alpha1
         alias: templatev1alpha1
+      - pkg: github.com/gardener/cert-management/pkg/apis/cert/v1alpha1
```

Comment:
I would like to align these in a separte follow-up pull request to:

- `gcertv1alpha1`, `gcertclientv1alpha1`
- `cmcertv1`, `cmcertclientv1`